tooltester/README.md

```diff
@@ -0,0 +1,70 @@
+Last tested with ansible 2.2.1.0
+# Master Setup:
+1.  Install the Jenkins pipeline suite
+2.  Install all plugins filtered with "blue ocean"
```

Comment:
Use a list of plugins in case some jerk decides to create an entirely different Jenkins plugin with "blue ocean in the title"